source/braille.py

```diff
@@ -2427,7 +2428,7 @@ def _dismissMessage(self, shouldUpdate: bool = True):
 			self.update()
 
 	def handleGainFocus(self, obj: "NVDAObject", shouldAutoTether: bool = True) -> None:
-		if not self.enabled:
+		if not self.enabled or config.conf['braille']['mode'] == BrailleMode.SPEECH_OUTPUT.value:

```

Comment:
please fix up the single vs double quotations used in the rest of this file as requested earlier